source/install/heroku.rst

```diff
@@ -0,0 +1,31 @@
+Mattermost Preview using Heroku
+===============================
+
+Quickly set up Mattermost in Preview Mode to explore product functionality using Heroku.
+
+This configuration is not recommended for use in production, as it currently does not support email notifications. Also note that for persistent file storage AmazonS3 must be used, as local storage will be deleted periodically when the Heroku process restarts.
+
+Heroku Installation
+--------------------
+
+1. Sign up for a free account on `Heroku <https://www.heroku.com/>`_
+
+2. Click the |HerokuDeploy|_ button
+
+.. |HerokuDeploy| image:: https://www.herokucdn.com/deploy/button.svg
+.. _HerokuDeploy: https://heroku.com/deploy?template=https://github.com/mattermost/mattermost-heroku
+3. (Optional) Adjust the settings in Heroku
```

Comment:
Maybe just mention that these fields autocomplete but if you'd like to customize your setup then you can optionally edit these fields.